### DISCUSSION TITLE
pretty-url breaking the edit link reference

### DISCUSSION THREAD

**Discussion by laujonat** [srid/neuron#642] [General]
Hi, I am running into a little trouble with the edit url in my `.dhall` file pointing at `https://www.cerveau.app/-/blah/neuron-repo/some-branch`

Is there a setting that addresses the generated edit links from `neuron gen --pretty-url`? The edit link breaks because `website.com/hello_world.md` becomes `website.com/hello_world`, resulting in an invalid zettel.   

Yes, I can just change the URL manually, but I want it to just work lol. I am aware we can use Javascript as a workaround to strip the `.md` of each link, but I was curious if there was a flag or setting I can use to avoid that.  

Sorry ahead of time if this is a duplicate, I couldn't find a issue for this question.  Thanks! :D

**Comment by srid** [comment]
No feature exists to do this. I think JavaScript approach would be best here.

**Reply by laujonat** [reply]
noted, thanks!
